lib/jsmidgen.js

```diff
@@ -398,7 +398,7 @@ var Midi = {};
 		// add the number of tracks (2 bytes)
 		bytes += Util.codes2Str(Util.str2Bytes(trackCount, 2));
 		// add the number of ticks per beat (currently hardcoded)
-		bytes += File.HDR_SPEED;
+		bytes += this.HDR_SPEED;
```

Comment:
We should be able to do something similar to the line above where `trackCount` is output to the MIDI file.